org.ekstep.keyboard-1.1/renderer/style/style.css

```diff
@@ -5,7 +5,7 @@
   left: 0;
   right: 0;
   bottom: 0;
-  height: 50%;
+  height: 42%;
```

Comment:
@JagadishPujari height hardcoded to 42% did you verified this in all resolutions?

Reply:
This player half height we are using for the keyboard so hardcoded, and verified for all devices and working fine
CC: @vinukumar-vs